### ISSUE TITLE
'includePaths' does not work with LESS

### ISSUE THREAD

**Issue by taras-d** [angular/angular-cli#7083]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.2.2
node: 6.11.0
os: linux x64

### Repro steps.
1. Create project with less styles `ng new [NAME] --style=less`
2. Create `styles/variables.less` directory in `src` folder
3. Update styles config in `.angular-cli.json`
```json
"styles": [
    "styles/variables.less"
],
"stylePreprocessorOptions": {
    "includePaths": [
        "styles"
    ]
},
```
4. Import variables in component styles
```less
@import 'variables';

div {
    color: @text-color;
}
```

### The log given by the failure.
```
ERROR in ./src/app/app.component.less
Module build failed:

@import 'variables';
^
Can't resolve './variables.less' in '/home/user/Desktop/ng-less/src/app'
      in /home/user/Desktop/ng-less/src/app/app.component.less (line 1, column 0)
 @ ./src/app/app.component.ts 18:17-48
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi webpack-dev-server/client?http://localhost:4242 ./src/main.ts
```

### Desired functionality.
`includePaths` should work like less `paths` option

### Mention any other details that might be useful.
The steps were reproduced with `scss` styles and it worked correctly

**Comment by grizzm0** [comment]
It was not included as it was blocked by https://github.com/webpack-contrib/less-loader/pull/75

**Comment by taras-d** [comment]
Is there any workaround?

It becomes inconveniently to use relative imports in case of deep folder structure
(for example `@import '../../../../../variables`)

**Comment by filipesilva** [comment]
This is not a bug but a feature request. Less support was never included in the functionality (https://github.com/angular/angular-cli/pull/4003). 

Now that it's possible, I'd like to integrate it. PRs are welcome, but bear in mind it should use the built-in LESS resolver and not the webpack resolver.

**Comment by JPeer264** [comment]
I could implement it.

**Comment by JPeer264** [comment]
@filipesilva shortly v`1.3.0` was released, but it seems that the feature of #7212 is not inside of this release (I cross-checked it with the output in my local node_modules). Or are the releases not of the latest commit of the master?

**Comment by filipesilva** [comment]
@JPeer264 your PR was released in `v1.4.0-beta.0` and up. We only add features to a release in `-beta` releases so it did not get into 1.3.
